SilMock/Google/Service/Directory/UsersAliasesResource.php

```diff
@@ -42,8 +42,18 @@ public function delete($userKey, $alias)
             throw new Exception(self::ACCOUNT_DOESNT_EXIST . $userKey, 201407101645);
         }
 
-        // Get all the aliases for that user
         $sqliteUtils = new SqliteUtils($this->dbFile);
+        // Get all the user records for that user
```

Comment:
```suggestion

        // Get all the user records for that user
```

Reply:
Do we want to PSR2 check the codebase on PR submit?

Reply:
PSR-12, sure. PSR-2, no. We have 4 lines of code which conflict between the two standards. They are multi-line if statements.